reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb

```diff
@@ -0,0 +1,748 @@
+{
```

Comment:
<p>Should we mention that the rl notebooks are also using Experimentation Mode? To avoid different terms are used in RL and Bandit notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>